rmgpy/reaction.py

```diff
@@ -636,13 +636,31 @@ def get_equilibrium_constant(self, T, type='Kc', surface_site_density=2.5e-05):
         dN_surf = number_of_surface_products - number_of_surface_reactants # change in mols of surface spcs
         dN_gas = number_of_gas_products - number_of_gas_reactants # change in mols of gas spcs
 
+        # Determine the multiplication factor of the binding site^(-stoichiometric coefficient)
+        # (only relevant for reactions involving multidentate adsorbates)
+        unique_species = []
+        sigma_nu = 1
+        for product in self.products:
+            if len(product.molecule) != 0:
```

Comment:
I'd probably wrap it in a try/catch instead of putting this check. It's (during a 'typical' RMG run) always going to pass this check, so checking it is a waste of time.

Reply:
Do we need a try/catch, even? Are there cases where a reaction would have either no products or no reactants?

Reply:
Why do we need this case for the `test_equilibrium_constant_kc` where the molecule is removed or what is the purpose of it? If we get rid of this part of the test, we could get rid of the check.